source/install-desktops/install-desktop-app.rst

```diff
@@ -0,0 +1,103 @@
+.. _install-desktops-install-desktop-app:
+
+Install Desktop App
+===================
+
+No we will go through installing the provided `bc_desktop`_ app as well as
+configuring it work at your center.
+
+#. We will build this app in our previous web applications setup directory
+   (discussed in :ref:`install-apps`):
+
+   .. code-block:: sh
+
+      cd ~/ood/src/apps/
+
+#. By default this app is not built and installed with all the other required
+   OOD apps. You will need to build this app explicitly with:
+
+   .. code-block:: sh
+
+      scl enable rh-ruby22 nodejs010 git19 -- rake build:bc_desktop
+
+   .. note::
+
+      The Desktop App is an **optional** app and will not be built and
+      installed when you call:
+
+      .. code-block:: sh
+
+         scl enable rh-ruby22 nodejs010 git19 -- rake
+         sudo scl enable rh-ruby22 -- rake install
+
+      Instead you need to explicitly build and install it to prevent you from
+      installing an app that is not properly configured for your users:
+
+      .. code-block:: sh
+
+         scl enable rh-ruby22 nodejs010 git19 -- rake build:bc_desktop
+         sudo scl enable rh-ruby22 -- rake install:bc_desktop
+
+#. We now must configure desktop apps for each cluster you want this to run
+   under in the following directory ``local/``
+
+   .. code-block:: sh
+
+      # Make the `local/` directory if it doesn't already exist
+      mkdir local
+
+      # Change our working directory to `local/`
+      cd local
+
+#. For each cluster we want to launch desktop on we will need a corresponding
+   YAML configuration file (e.g., ``cluster1.yml``, ``cluster2.yml``, ...) that
```

Comment:
I am fine with renaming it. The `.erb` is optional although we don't currently have a use-case for it at OSC in the form configuration.